docs/release_notes/release_notes.md

```diff
@@ -8,42 +8,53 @@ has_children: true
-    * Added a new `License Events` panel to show all Palo Alto license-related events.
+* ### Office 365 alert and dashboard improvement
+    * Updated the alerts and dashboard to use the new o365:service:healthIssue sourcetype (as the o365:service:status sourcetype is retired)
+    * Added `Login by location` map in the `Office 365` dashboard.
+    * Added `authentication_method` and `user_type` fields in the o365 login related alerts and panels.
 
-* ### Palo Alto Firewall DNS Sinkhole improvement
-    * Added `url` field in the `Palo Alto DNS Sinkhole` alert and forensic searches.
+* ### Active Directory alert and dashboard improvement
+    * Added new alert `AD - Multiple Password Changes in Short Time Period` to find the mulitple password change in the short time period.
+    * Added new alert `AD - Bulk User Creation or Deletion` to find the bulk user creation or deletion.
+    * Added 
+    * Enhanced the `AD - Password Change Outside Working Hour` alert and panel to show adiitional improtant fields.
 
-* ### DNS Tracker dashboard improvement.
-    * Added hostname details for any IP field to easily identify the machine based on the information available in Device Inventory lookup.
-    * Separated panels to show information about `actual hosts making DNS requests` vs `DNS servers making DNS requests to other internal DNS servers` vs `DNS servers requesting external DNS servers`. 
-        * This will give more clarity about who is the actual requester, how well each internal DNS server is performing, any malicious behavior by a client/source, lots of malicious/incorrect responses received from a specific external DNS server, etc.
+* ### G Suite new alert
+    * Added new alert `G Suite - Multiple Password Changes in Short Time Period` to find the mulitple password change in the short time period.
+    * Added new alert `G Suite - Bulk User Creation or Deletion` to find the bulk user creation or deletion.
+    * Added `User Created` and `User Deleted` panels in the G Suite dashboard.
 
 
 * ### Bug Fixes
-    * Fixed a drilldown issue for the `Antivirus` panel in the `Overview` dashboard.
-
-    * Fixed the token name to populate the `PowerShell Script Execution Error` panel of `Microsoft 365 Defender ATP Audit` dashboard.
+    * Fixed a drilldown issue for the `Login Details` panel in the `VPN` dashboard.
 
-    * Fixed the `Linux/Unix` dashboard to show results when some fields in the data are not present.
+    * Fixed the source value in the `cs_sysmon macro` macro.
 
-* ### Cloud Compatibility Issue Fixed
-   * Fixed the cloud vetting issue to make addon cloud compatible by validating that the App only makes requests to secure HTTPS URLs.
 
+## Upgrade Guide from 2.1.0 to 2.2.0
 
-## Upgrade Guide from 2.0.0 to 2.1.0
+* User need to enable the `win_listening_ports` scripted input from the Splunk Add-on for Windows addon to populate the `Listening Ports on Host` panel of `Windows Reports` dashboard.
```

Comment:
the Splunk Add-on for Windows addon to
->
the Splunk Add-on for Windows

Reply:
replaced `the Splunk Add-on for Windows addon` -> `the Splunk Add-on for Windows`